atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -627,7 +627,7 @@ private void verifyColumnRanges(Transaction readOnlyTransaction) {
                     if (rangesToRows.get(range) != null) {
                         rangesToRows.get(range).add(row);
                     } else {
-                        rangesToRows.put(range, ImmutableList.of(row));
+                        rangesToRows.put(range, Lists.newArrayList(row));
```

Comment:
could simplify the whole if/else to:

```
                    rangesToRows.computeIfAbsent(range, ignored -> Lists.newArrayList()).add(row);
```

Reply:
I'm happy either way. @sandorw?

Reply:
Going to keep this for now for the sake of merging and getting the release out. I'll bundle it with another PR which is fixing some other code style in the same class (https://github.com/palantir/atlasdb/pull/3149).